reduceEpsilons.py

```diff
@@ -160,7 +165,20 @@ def main(tolerance, step, username):
         print("step - float, increment +/- 10x around known beta (maximum 0.00001)")
         print("username - the user who will be running the calibration on the cluster")
         exit(0)
-
+        # Parse the parameters
+        configuration = sys.argv[1]
+        studyId = int(sys.argv[2])
+
+        # Parse out the zero filter if one is provided, otherwise default True
+        zeroFilter = True
+        if len(sys.argv) == 4:
+            if sys.argv[3] == "0":
+                zeroFilter = False
+                print("Zero filter disabled")
+            if sys.argv[3] not in ["0", "1"]:
+                print("Flag for filter must be 0 (false) or 1 (true)")
+                exit(1)
```

Comment:
Ditto above, this looks like a carry over from createBetaMap.py which allows for zero filtering to be turned on/off, but since this script is acting upon the beta map filtering is not needed.